bin/reth/src/node/mod.rs

```diff
@@ -123,6 +124,13 @@ pub struct Command<Ext: RethCliExt = ()> {
     #[arg(long, value_name = "SOCKET", value_parser = parse_socket_address, help_heading = "Metrics")]
     metrics: Option<SocketAddr>,
 
+    /// Add a secondary node
+    ///
+    /// Configures the ports of the node to avoid conflicts with the defaults.
+    /// This is useful for running multiple nodes on the same machine.
+    #[arg(long)]
+    secondary: bool,
```

Comment:
imo this is very useful to setup two instances,

I wonder if we can make this even more scalable, so we can have three or more,

but --secondary should be the most useful.

the docs should list what ports are being used so they're displayed in help message